blazor/datagrid/template-pdf-export.md

```diff
@@ -0,0 +1,592 @@
+---
+layout: post
+title: Pdf Export with Templates in Blazor DataGrid Component | Syncfusion
+description: Checkout and learn here all about Template Pdf Export in Syncfusion Blazor DataGrid component and much more details.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# PDF Exporting with Templates
+
+### Export template columns
+
+PDF export provides an option to export template columns of the DataGrid by defining the [IncludeTemplateColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.PdfExportProperties.html#Syncfusion_Blazor_Grids_PdfExportProperties_IncludeTemplateColumn) of [PdfExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.PdfExportProperties.html) as true. In the following sample, the CustomerID column is a template column. The template values cannot be directly exported into the cells. To customize the values of template columns in the PDF document, you must use [PdfQueryCellInfoEvent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_PdfQueryCellInfoEvent).
```

Comment:
use this as second paragraph: The PDF export functionality allows you to export Grid columns that include images, hyperlinks, and custom text to an PDF document.

Also, please follow the instructions that are provided for excel export,
https://github.com/syncfusion-content/blazor-docs/pull/4139/files#r1531792560

Reply:
Hi @Venkat-Ayothi ,

I have done the review changes could you please check and let me know.

Regards,
R.Dhivya